tests/Our.Umbraco.Ditto.Tests/PublishedContentTests.cs

```diff
@@ -55,6 +55,7 @@ public void Property_Returned()
         }
 
         [Test]
+        [ExpectedException(typeof(System.InvalidOperationException))]
```

Comment:
@mattbrailsford I hit a snag here, where the `SimpleModel` has a string property and is assigned an `IPublishedContent` ... previously Ditto did a `ToString()` with it, (well, Umbraco's `TryConvertTo` did it) ... Now it throws an exception, (as it tried to construct a `string` object).